### PR TITLE
Performance configuration

### PR DESCRIPTION
**Pull request by leoncx**
<h1> Pull Request Template </h1>

<h2> Description </h2>

* Change the options display to configure Centreon Engine :  Broker Module Options.
* Add configuration options for Centreon Engine Macros filtering on configuration page
* Add configuration options for Centreon Engine Event send filtering on configuration page
* Add configuration options for Centreon Broker SQL Output for number of connections

Fixes MON-3662, MON-3625

<h2> Type of change </h2>

- [ ] Patch fixing an issue (non-breaking change)
- [X] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [ ] 18.10.x
- [X] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [X] I followed the **coding style guidelines** provided by Centreon
- [X] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [X] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

<h5> Centreon team only </h5>

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [X] I have made sure that **unit tests covers 80%** of the code written for the story.
- [X] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: www/install/sql/centreon/Update-DB-18.10.4_to_19.04.1.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-- Add new configuration for Centeron Broker : Use multiple thread to connect to database
+INSERT INTO `cb_field` (`cb_field_id`, `fieldname`, `displayname`, `description`, `fieldtype`, `external`) VALUES
+(75, 'connections_count', 'Connections count', 'Number of opened connection to the database.', 'int', NULL),
+(76, 'enable_command_cache', 'Enable command cache', 'Cache the executed command for better insert performance. Warning: This options use more memory.', 'radio', NULL);
```
Comment (bouda1) [inline www/install/sql/centreon/Update-DB-18.10.4_to_19.04.1.sql]: The cache filter is no more configurable in centreon-broker. This configuration part must be removed.

File: www/include/configuration/configNagios/DB-Func.php [hunk 2]
```diff
@@ -219,6 +219,34 @@ function insertNagiosInDB()
...
+ *  Enable 0|1 if the bit is activate
+ *
+ * if found the bit -1 (all) or 0 (none) activate return the value
+ *
+ * @param array $list The POST QuickForm table
+ * @return int The bitwise
+ */
+function calculateBitwise($list)
+{
+    $bitwise = 0;
```
Comment (lpinsivy) [inline www/include/configuration/configNagios/DB-Func.php]: Should not be in lowerCamelCase (https://github.com/centreon/centreon/tree/master/doc/coding-style/php#php)

**Comment by lpinsivy** [comment]
Form and exported values (centengine.cfg) work great!

File: www/include/configuration/configNagios/DB-Func.php (hunk 2 above)
Comment (leoncx) [reply www/include/configuration/configNagios/DB-Func.php]: bitwise is one word, so calculateBitwise is lowerCamelCase

**Comment by sc979** [comment]
As part of this PR is already reviewed and approved, I close this PR.
But PLEASE don't delete the branch !!!

**Pull request closed without merging**